### PR TITLE
Welsh localisation

### PR DESCRIPTION
**Pull request by Cymrodor**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the unit tests with `gradlew app:connectedGmsDebugAndroidTest` to make sure you didn't break anything (with an emulator running or phone connected)

- [ ] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by jaydeetay** [comment]
Diolch yn fawr!

**Comment by Cymrodor** [comment]
Many thanks for merging these. I'm not a developer, so the requirement to test the changes was beyond my skillset.

**Comment by jaydeetay** [comment]
Hi. If you like I can send you a link to the app to test it. I'd like to
bundle this with a few other changes and get it out soon.
I'm super excited about these translations...my own welsh education stopped
age 14 with me barely able to count to 10 and wish you good morning.
Regretted that ever since. Let me know how to credit you in the release
notes.

John

**Comment by Cymrodor** [comment]
I would appreciate that. It's been a few months since I did the translations, so I'm not sure whether anything may have changed, e.g. any new strings related to Neowise (old by now?!). Seeing the app is always a great help to proof read. Diolch yn fawr iawn.

**Comment by Cymrodor** [comment]
You can credit me either as username Cymrodor or real name Aled Powell, following whichever is the norm for consistency.

**Comment by jaydeetay** [comment]
Hi Aled - I've run into a bit of a problem. My fault for not looking too closely at the pull request.  The content of the strings.xml file isn't what I was expecting.  If you look at the other languages it's in "xml" - a particular format needed to build the app. The file you submitted doesn't follow the same format. I'd reformat it myself, but I'm not quite sure how to interpret it. Can you check if it's the file you intended to submit?

John

**Comment by jaydeetay** [comment]
Hi Aled,
Finally I've got a test version up and running.  You can download it here
if you want to take it for a spin and see if it looks right:

https://play.google.com/apps/test/RQjmMu2nXvg/ahAJEhp-kMvkA7GO4D7a-R_eosrBCm3x54_nLaeFH0AUPtjaMuaT4sBdKXuyvk02WdV-JSLMuCL-DiTsi71gEproGh

John

File: app/src/main/res/values-cy/help.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+&lt;b&gt;Clodrestr&lt;/b&gt;&lt;br/&gt;
+&lt;i&gt;Peirianneg, Dylunio a Rheolaeth Cynnyrch&lt;/i&gt;&lt;br/&gt;
+Brent Bryan, Dominic Widdows, Hector Ouilhet, James Powell,
+John Taylor, Kevin Serafini, Lior Ron&lt;br/&gt;&lt;br/&gt;
+
+&lt;i&gt;Cyfranwyr Cod Agored&lt;/i&gt;&lt;br/&gt;
+Christopher Wilcox, PhD&lt;br/&gt;&lt;br/&gt;
+
+&lt;i&gt;Lleoleiddiad Cymraeg&lt;/i&gt;&lt;br/&gt;
```
Comment (jaydeetay) [inline app/src/main/res/values-cy/help.xml:110]: Glad to see you put this in @Cymrodor 
I'm going to try to remember to suggest this for all translators.

**Pull request merged**